Formula/q.rb

```diff
@@ -3,8 +3,8 @@ class Q < Formula
 
   desc "Run SQL directly on CSV or TSV files"
   homepage "https://harelba.github.io/q/"
-  url "https://github.com/harelba/q/archive/2.0.18.tar.gz"
-  sha256 "db4f90602d3c45395675fa09b94293078f0ff693a94ec5fb23ae7041492fcebd"
+  url "https://github.com/harelba/q/archive/2.0.19.tar.gz"
+  sha256 "cd4c60923bc40f53d974b54849f76096bf9901407c618cd0a3ccbc322aacc97d"
   license "GPL-3.0-only"
```

Comment:
@harelba this should be `GPL-3.0-or-later`, right? (as I read thru the file header)

Reply:
true. Fixing. thanks.